docs/content/en/docs/concepts/architecture/components/KLTcomponents.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Lifecycle Toolkit Components
+linktitle: KLT Components
+description: Basic understanding of Keptn Lifecycle Toolkit Components
+weight: 80
+cascade:
+---
+
+### Keptn Lifecycle Toolkit Components
+
+The Keptn Lifecycle Toolkit consists of two main components:
+
+* Keptn Lifecycle Operator, splits into two separate operators:
+  * Lifecycle-Operator
+  * Metrics-Operator
+* Keptn Lifecycle Scheduler
+
+```mermaid
+graph TD;
+    KLTComponents-->Operators;
+    KLTComponents-->Scheduler
+   Operators-->Lifecycle-Operator
+   Operators-->Metrics-Operator
+
+style KLTComponents fill:#006bb8,stroke:#fff,stroke-width:px,color:#fff
+style Operators fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+style Scheduler fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+style Lifecycle-Operator fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+style Metrics-Operator fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+
+```
+
+**Keptn's Lifecycle Operator** is
+a Kubernetes [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
+that automates the deployment and management
+of Keptn's Lifecycle Controller components in a Kubernetes cluster.
+The Keptn Lifecycle Operator contains several controllers for **Keptn CRDs**
+and a **Mutating Webhook**.
+
+Here's a brief overview:
+
+**Keptn CRDs:** The Keptn Lifecycle Operator contains
+several controllers that are responsible for managing
+and reconciling different types of Keptn CRDs, some of them are
+Project Controller,Service Controller,Stage Controller,and more.
+
+**Mutating Webhook:** It is used to automatically inject Keptn-specific labels
```

Comment:
```suggestion
**Mutating Webhook:** Automatically injects Keptn labels
```
You also can't hyphenate "Keptn" with another term.  But, in this case, I think just calling them Keptn labels is enough.

Reply:
ok mam, will keep in mind.